.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
```

Comment:
- [ ] Good job adding the linter for workflows. how ever, you're missing two other linter files which should be at the root of your directory.  Check the screenshot below and follow this [Link](https://github.com/microverseinc/linters-config/tree/master/html-css) to see the files and guidelines 

## Missing files
![image](https://user-images.githubusercontent.com/50990925/218893374-b6b1a108-e9d8-4804-a1e0-ecd20806db7c.png)

Reply:
Hello @Shedrack-Sunday 
Thank you for your review. I already made the corrections but now I need your help, please. I'm trying to push my branch on my repo, but I receive this message:

<img width="770" alt="image" src="https://user-images.githubusercontent.com/122847613/219140851-7d1bb655-813f-4089-93f3-589e02ff00e6.png">

 ! [rejected]        Hello-Project -> Hello-Project (non-fast-forward)
error: failed to push some refs to 'github.com:oovillagran/Hello-Microverse-project.git'

Please let me know what I supposed to do in this case.

Thank you in advance.

Reply:
Hello, @oovillagran Sorry for the late reply. Let me know if you still need help, and you can contact me via slack for a call

Reply:
Hi @Shedrack-Sunday, I already fixed it, thank you!!!.